source/dbo/Stored Procedures/Data Migration/uspMigrationAdditionalNotificationsByRegionAndYear.sql

```diff
@@ -14,21 +14,31 @@ AS
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_Hospital] tbh ON tbh.HospitalID = mn.NtbsHospitalId
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_PHEC] tbsp ON tbsp.TB_Service_Code = tbh.TB_Service_Code
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[PHEC] p ON p.PHEC_Code = tbsp.PHEC_Code
+		LEFT JOIN vwNotificationYear ny ON ny.NotificationYear = YEAR(mn.NotificationDate)
 	WHERE 
 	mn.GroupId IS NOT NULL
 	AND p.PHEC_Name = @Region
-	AND mn.NotificationDate >= '2017-01-01')
+	AND ny.Id >= -3
+	AND NOT EXISTS
+		(SELECT LegacyId
```

Comment:
Now strictly speaking this shouldn't happen - if one of the group has been migrated into NTBS then they all have.  So we'd probably want to flag separately if we were going to migrate in any records that were somehow grouped with records already in NTBS. I THINK it's too unlikely to worry about.  We would find out during data cleaning anyway and be able to rectify it then.  So I think I would take this check out.